Module4/Assignment4/Assignment4.cpp

```diff
@@ -83,23 +238,34 @@ void printSortedStats(string sortType)
 }
 int main()
 {
-    int mergeSortInput[] = {5, 4, 3, 2, 1};
-    int mergeSortAccessCount = mergeSort(mergeSortInput, 5);
-    cout << "access count for merge sort was: " << mergeSortAccessCount << endl;
-    int bubbleSortInput[] = {5, 4, 3, 2, 1};
-    int bubbleSortAccessCount = bubbleSort(bubbleSortInput, 5);
-    cout << "access count for bubble sort was: " << bubbleSortAccessCount << endl;
-    int countingSortInput[] = {5, 4, 3, 2, 1};
-    int countingSortAccessCount = bubbleSort(countingSortInput, 5);
-    cout << "access count for counting sort was: " << countingSortAccessCount << endl;
 
-    int *bubbleOneData = makeRandomArray(8);
-    bubbleSortAccessCount = bubbleSort(bubbleOneData, 8);
-    cout << "Bubble array length of 8: " << bubbleSortAccessCount << endl;
-    delete[] bubbleOneData;
+    // Table headers:
+    cout << "Sort Algorithm"
+         << "\t";
+    cout << "Length"
+         << "\t";
+    cout << "Pass 1"
+         << "\t";
+    cout << "Pass 2"
+         << "\t";
+    cout << "Pass 3"
+         << "\t";
+    cout << "AVG" << endl;
     printSortedStats("bubble");
     printSortedStats("merge");
     printSortedStats("counting");
 
     return 0;
-}
\ No newline at end of file
+}
+/*SAMPLE OUTPUT
+Sort Algorithm	Length	Pass 1	Pass 2	Pass 3	AVG
+bubble  	    8	    11	    18	    7	    12
+bubble  	    32	    285	    246	    258	    263
+bubble  	    128	    3513	3879	4096	3829
+merge   	    8	    82	    80	    82	    81
+merge   	    32	    558	    560	    562 	560
+merge   	    128	    3248	3262	3268	3259
+counting	    8	    16	    16	    16	    16
+counting	    32	    64	    64	    64	    64
+counting	    128	    256	    256	    256	    256
+*/
```

Comment:
My general coding style is to always have a blank line at the end of the file (so just press return and you won't see the funny `-` sign here in github.